README.md

```diff
@@ -0,0 +1,72 @@
+
+# Lua.js
+
+[ALPHA] A non-crappy Lua 5.1 - 5.4 parsing and compiling library.
+
+## Description
+
+Lua.js is a modular and fast Lua interpreter written in TypeScript and JavaScript. It supports parsing and compiling Lua versions 5.1 to 5.4.
+
+## Features
+
+- Supports Lua 5.1 - 5.4
+- Modular design
+- Written in TypeScript for type safety
+- Fast and efficient parsing and compiling
+
+## Installation
+
+To install Lua.js, use npm:
+
+```bash
+npm install lua-js
```

Comment:
It's actually not published on NPM yet.
Plus, IIRC, the name "lua-js" is already taken in the NPM registry.
So I think it'd be better to install it from source, instead (for now).

So, something like that:
```
npm install git+https://github.com/bytexenon/Lua.js
```
would work

Reply:
ok